src/templates/pycontw-2020/contents/zh/venue.html

```diff
@@ -0,0 +1,109 @@
+{% extends 'contents/_venue.html' %}
+
+{% block title %}會場{% endblock title %}
+
+{% block map_info %}
+
+<h2>
+	<span>國立成功大學</span>
+	<span>國際會議廳</span>
+</h2>
+<p>701 台南市東區大學路 1 號</p>
+
+<script>window.VENUE_NAME = '國立成功大學 國際會議廳'</script>
+
+{% endblock map_info %}
+
+{% block content %}
+
+<h1>會場</h1>
+
+{{ block.super }}
+
+<h2>交通方式</h2>
+
+<h3>桃園機場（TPE）出發</h3>
+
+<h4>1. 桃園機場（TPE）到桃園高鐵</h4>
+
+<ul>
+  <li>
+    捷運
+    <ul>
+      <li>機場航廈站（A12 or A13）到高鐵桃園站（A18）</li>
+      <li><a href="https://www.tymetro.com.tw/tymetro-new/tw/_pages/travel-guide/timetable-search.php">時刻表</a></li>
+      <li>票價：NT$ 25</li>
+    </ul>
+  </li>
+  <li>
+    Uber
+    <ul>
+      <li>UberX Airport（NT$ 1,000 - 1,200）</li>
+      <li>UberXL Airport（NT$ 1,250 - 1,500）</li>
+      <li>UberBlack Airport（NT$ 1,500 - 1,800）</li>
+    </ul>
+  </li>
+</ul>
+
+<h4>2. 桃園高鐵站到台南高鐵站</h4>
+
+<ul>
+  <li>票價：NT$ 1,190</li>
+  <li><a href="https://www.thsrc.com.tw/tw/TimeTable/SearchResult">時刻表</a></li>
+</ul>
+
+<h4>3. 台鐵沙崙站到台南火車站</h4>
+
+<ul>
+  <li>沙崙站（4272）到台南站（4220）</li>
+  <li><a href="https://www.railway.gov.tw/tra-tip-web/tip/tip001/tip112/querybytime">時刻表</a></li>
+  <li>票價：NT$ 25</li>
+</ul>
+
+<h4>4. 台南火車站到成功大學</h4>
+
+<ul>
+  <li>走路大約 7 分鐘</li>
+  <li><a href="https://goo.gl/maps/w8uJKDGVhEbwwpMCA">Google Maps 路線</a></li>
+</ul>
+
+<h3>小港機場（KHH）出發</h3>
+
+<h4>1. 捷運小港站到高雄車站</h4>
+
+<ul>
+  <li>小港站（R3）到高雄車站（R11）</li>
+  <li>票價：NT$ 30</li>
+  <li><a href="https://www.krtc.com.tw/Guide/timetable">時刻表</a></li>
+</ul>
+
+<h4>2. 台南火車站到成功大學</h4>
+
+<ul>
+  <li>走路大約 7 分鐘，約 550 公尺</li>
+  <li><a href="https://goo.gl/maps/w8uJKDGVhEbwwpMCA">Google Maps 路線</a></li>
+</ul>
+
+<h1>專業課程與衝刺開發場地</h1>
+
+{% block map_info %}
```

Comment:
You redefined `map_info` block. Block should be unique.

Reply:
@yychen : May the Dev team help us to add another `map_info` for Goodideas-Studio? Thank you.

Reply:
I've added something. Can you allow maintainer to push to this branch? Thanks!

Reply:
@yychen: Thanks for your help!